Spelling non-API change

jsoref opened the pull request.

jsoref commented on maven-compat/src/main/java/org/apache/maven/repository/MetadataGraphNode.java:
```diff
@@ -57,7 +57,7 @@ public MetadataGraphNode addIncident( MetadataGraphNode node )
     }
 
-    public MetadataGraphNode addExident( MetadataGraphNode node )
+    public MetadataGraphNode addExcident( MetadataGraphNode node )
```
This would be an api change. I'm not even sure if this is a correct word.
I'm fairly confident that you are using two spellings for the same concept.

jsoref commented on maven-compat/src/main/java/org/apache/maven/repository/MetadataResolutionRequest.java:
```diff
@@ -108,14 +108,14 @@ public MetadataResolutionRequest setLocalRepository( ArtifactRepository localRep
     }
 
-    public List<ArtifactRepository> getRemoteRepostories()
+    public List<ArtifactRepository> getRemoteRepositories()
```
This is probably an api change.
I can provide deprecation markers and/or split this into a distinct PR (applies for all similar items)

jsoref commented on maven-core/src/main/java/org/apache/maven/project/ProjectSorter.java:
```diff
@@ -105,7 +105,7 @@ public ProjectSorter( Collection<MavenProject> projects )
         }
 
-        for ( Vertex projectVertex : dag.getVerticies() )
+        for ( Vertex projectVertex : dag.getVertices() )
```
this change stands alone, I can't find anything else that uses this spelling, but lots of things use the corrected spelling....

jsoref commented on maven-core/src/main/java/org/apache/maven/repository/DelegatingLocalArtifactRepository.java:
```diff
@@ -61,7 +61,7 @@ public void setIdeWorkspace( LocalArtifactRepository localRepository )
     }
 
-    public LocalArtifactRepository getIdeWorspace()
+    public LocalArtifactRepository getIdeWorkspace()
```
This would be an api change

jsoref commented on maven-core/src/test/java/org/apache/maven/lifecycle/internal/stub/LifecycleExecutionPlanCalculatorStub.java:
```diff
@@ -88,7 +88,7 @@
 
 
-    public int getNumberOfExceutions( ProjectBuildList projectBuildList )
+    public int getNumberOfExecutions( ProjectBuildList projectBuildList )
```
I'm hoping this isn't really a public api, but if it is...

jsoref commented on maven-model-builder/src/test/java/org/apache/maven/model/interpolation/AbstractModelInterpolatorTest.java:
```diff
@@ -67,7 +67,7 @@ protected void assertProblemFree( SimpleProblemCollector collector )
     }
 
-    protected void assertColllectorState( int numFatals, int numErrors, int numWarnings,
+    protected void assertCollectorState( int numFatals, int numErrors, int numWarnings,
```
I guess this is an api

michael-o commented:
Hi Josh, thanks for the PR. I have seen that you did the same for Tomcat. We should split this into two parts:

1. non-API change
2. API change with deprecations and proper docs

Please create two tickets, I wil take care of.

jsoref commented:
@michael-o split -- please follow-up in #103 as that's a messier kettle

michael-o commented:
@jsoref I am somewhat confused by the new PRs. I wanted two distinct PRs. Can you do that?

jsoref commented:
@michael-o : this PR is roughly "non-API change". I split off #101 for the API change (and included deprecation, but not docs, that's too much to ask).
#102 is a bug fix, I think, so I split it out.
#103 isn't really finished, because well, I'm not sure what the right word is.

michael-o commented on maven-core/src/main/resources/META-INF/plexus/components.xml:
```diff
@@ -122,8 +122,8 @@ under the License.
           <field-name>_cipher</field-name>
         </requirement>
         <requirement>
-          <role>org.sonatype.plexus.components.sec.dispatcher.PasswordDecryptor</role>
```
This is a typo in [Plexus Sec Dispatcher](https://github.com/sonatype/plexus-sec-dispatcher/blob/master/src/main/java/org/sonatype/plexus/components/sec/dispatcher/PasswordDecryptor.java#L25) we can't fix here.

michael-o commented on maven-core/src/main/resources/META-INF/plexus/components.xml:
```diff
@@ -122,8 +122,8 @@ under the License.
         </requirement>
         <requirement>
-          <role>org.sonatype.plexus.components.sec.dispatcher.PasswordDecryptor</role>
-          <field-name>_decryptors</field-name>
```
Same here.

michael-o commented:
I am about to mere this PR, but for one commit which has to be undone. Then I will check the rest.

jsoref commented:
sonatype/plexus-sec-dispatcher#4

michael-o commented:
You have renamed to `mngt`, but missed to rename physical directories also.

michael-o commented:
Can you rebase your changes and squash into one commit? I want to pull them in with [MNG-6146](https://issues.apache.org/jira/browse/MNG-6146?src=confmacro).

jsoref commented:
done

michael-o commented:
Testing your PR, two issues:

1. Please resolve the marked conflicts from above
2. One test fails because you missed to rename a directory:
-------------------------------------------------------------------------------
Test set: org.apache.maven.project.PomConstructionTest
-------------------------------------------------------------------------------
Tests run: 122, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 4.906 sec <<< FAILURE! - in org.apache.maven.project.PomConstructionTest
testInterpolationWithSystemProperty(org.apache.maven.project.PomConstructionTest)  Time elapsed: 0.031 sec  <<< ERROR!
org.apache.maven.project.ProjectBuildingException: 1 problem was encountered while building the effective model
[FATAL] Non-readable POM D:\Projekte\maven\maven-core\src\test\resources-project-builder\system-property-interpolation: D:\Projekte\maven\maven-core\src\test\resources-project-builder\system-property-interpolation (Das System kann die angegebene Datei nicht finden) @ 
 for project  at D:\Projekte\maven\maven-core\src\test\resources-project-builder\system-property-interpolation
	at org.apache.maven.model.building.DefaultModelProblemCollector.newModelBuildingException(DefaultModelProblemCollector.java:197)
	at org.apache.maven.model.building.DefaultModelBuilder.readModel(DefaultModelBuilder.java:598)
	at org.apache.maven.model.building.DefaultModelBuilder.build(DefaultModelBuilder.java:273)
	at org.apache.maven.project.DefaultProjectBuilder.build(DefaultProjectBuilder.java:161)
	at org.apache.maven.project.DefaultProjectBuilder.build(DefaultProjectBuilder.java:119)
	at org.apache.maven.project.PomConstructionTest.buildPom(PomConstructionTest.java:1861)
	at org.apache.maven.project.PomConstructionTest.buildPom(PomConstructionTest.java:1831)
	at org.apache.maven.project.PomConstructionTest.testInterpolationWithSystemProperty(PomConstructionTest.java:1521)

Please update.

jsoref commented:
Updated

michael-o commented:
Testing your changes now: https://builds.apache.org/job/maven-3.x-jenkinsfile/job/MNG-6146/

jira-importer commented:
Resolve #7298
